### ISSUE TITLE
SF-85, SF-85P, and SF-86c

### ISSUE THREAD

**Issue by patrickmcconnell** [18F/bpa-opm-eqip#35]
# Question/Comment on e-QIP Prototype RFQ

## Name and affiliation

Patrick McConnell, SemanticBits Agile BPA Manager

## Section of RFQ documents

eQIP PWS

## Question/Comment

Is it in-scope during the base period or any option periods to provide digital support to SF-85, the SF-85P, and the SF-86c, in addition to SF-86 specified in the PWS?

**Comment by billygriffin** [comment]
No, the scope includes only the SF-86. However, this product is intended to scale in the future to accommodate all other forms that are part of the e-QIP system or needed for investigation purposes. The prototype is expected to be flexible enough that it can accommodate more forms in the future.
